main.go

```diff
@@ -17,3 +29,19 @@ func main() {
 	t := tree.Tree(lines, "/")
 	t.Print(0, nil)
 }
+
+func handleArguments() {
```

Comment:
I did, and ended up removing it.

1. Flag doesn't handle `--` for whatever reason.
2. https://github.com/jessevdk/go-flags felt heavy-handed for this.

thoughts?